src/imaging.py

```diff
@@ -41,18 +41,23 @@
                     help='Latitude of location. Default = +44d13m29s')
 parser.add_argument('--longitude', type=str, nargs='?', const='-76d29m52s', default='-76d29m52s',
                     help='Longitude of location. Default = -76d29m52s')
-parser.add_argument('--image_width', type=int, nargs='?', const=0, default=8,
+parser.add_argument('--image_width', type=int, nargs='?', const=8, default=8,
                     help='Height of image in degrees (int). Default = 8')
-parser.add_argument('--image_height', type=int, nargs='?', const=0, default=8,
+parser.add_argument('--image_height', type=int, nargs='?', const=8, default=8,
                     help='Width of image in degrees (int). Default = 8')
+parser.add_argument('--sun_alt', type=float, nargs='?', const=400, default=400,
+                    help='Starting altitude of the sun, manual input for testing. Ex. 22.5. Default = 400')
+parser.add_argument('--sun_az', type=float, nargs='?', const=400, default=400,
```

Comment:
I think maybe we give the option of taking in the image starting point instead of sun starting point -- this will allow us to take a large image of the entire landscape if we want (with less math too).  And when we want to take an image of the sun we can just use the normal getSunPosition part